Update prompt version message doc video.

nahuel-borda opened the pull request:
https://github.com/MagnivOrg/prompt-layer-docs/assets/22693874/2aba2b9c-0e41-4f53-a1ee-3e175a8cfc55

nahuel-borda committed:
Update prompt-version-commit-message.mp4

jzone3 commented:
Nitpick but, why do you click "Save" without actually changing the template in this video? Also, why is there no User message? 

Also show the commit message at the end of the video :)

nahuel-borda committed:
Make commit message video more realistic

nahuel-borda commented:
> Nitpick but, why do you click "Save" without actually changing the template in this video? Also, why is there no User message?
> 
> Also show the commit message at the end of the video :)

Great suggestions. I implemented all of them. Just updated the PR video.

jzone3 commented:
LGTM
